rts/lra/lra-test/lra-arquillian-extension/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>lra-test</artifactId>
+    <groupId>org.jboss.narayana.rts</groupId>
+    <version>5.9.8.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>lra-arquillian-extension</artifactId>
```

Comment:
could you please rename this for `lra-test-arquillian-extension`. I know it's a little bit picky from but it's nicer in listing of the module names. And as it's for tests it gives me sense.